packages/core/util/tests/util-specs.js

```diff
@@ -68,6 +122,46 @@ describe('util', () => {
 		});
 	});
 
+	describe('mergeClassNameMaps', () => {
+		const baseMap = {
+			'class-base-only': 'real-class-base-only',
+			'class-shared': 'real-class-shared-base',
+			'class-shared-another': 'real-class-shared-another-base'
+		};
+		const additiveMap = {
+			'class-shared': 'real-class-shared-additive',
+			'class-shared-another': 'real-class-shared-another-additive',
+			'class-additive-only': 'real-class-additive-only'
+		};
+
+		test('should return a base map if an additive map is not given', () => {
+			const expected = mergeClassNameMaps(baseMap);
+
+			expect(expected['class-base-only']).toEqual('real-class-base-only');
```

Comment:
Can we just use toEqual? like expect(expected).toEqual(baseMap).

Reply:
It is possible only for when an additive map is not given since other cases need to be checked with merged result. And, I want to make `expect`s format as same as each other. WDYT?